src/renderer/qss/js/qssSettingButtons.js

```diff
@@ -118,4 +118,120 @@
+                args: ["{that}.dom.image", "{that}.model.item.schema.image", "{arguments}.0"]
+            },
+            hideTitle: {
+                funcName: "gpii.qss.volumeButtonPresenter.hideTitle",
+                args: ["{that}.dom.title", "{arguments}.0"]
+            },
+            changeButtonColor: {
+                funcName: "gpii.qss.volumeButtonPresenter.changeButtonColor",
+                args: ["{that}.container", "{that}.options.styles.redButton", "{arguments}.0"]
+            }
+        }
+    });
+
+
+
+    /**
+     * Show or hide the title of the button.
+     * @param {jQuery} titleElem - The jQuery object corresponding to the title of the button.
+     * @param {Number} value - The state of the button.
+     */
+    gpii.qss.volumeButtonPresenter.hideTitle = function (titleElem, value) {
+        if (!value) {
+            titleElem.hide();
+        } else {
+            titleElem.show();
+        }
+    };
+
+    /**
+     * If available in the setting's schema, shows the specified image for the button when the value of setting is 0.
+     * @param {jQuery} imageElem - The jQuery object corresponding to the image of the button.
+     * @param {String} image - The path to the image.
+     * @param {Number} value - The state of the button.
+     */
+    gpii.qss.volumeButtonPresenter.renderImage = function (imageElem, image, value) {
+        if (image && !value) {
+            var maskImageValue = fluid.stringTemplate("url(\"%image\")", {
+                image: image
+            });
+
+            // use a mask image to avoid having 2 different images (one for when the
+            // button is activated and one when it is not)
+            imageElem.css("mask-image", maskImageValue);
+            imageElem.show();
+        } else {
+            imageElem.hide();
+        }
+    };
+
+    /**
+     * Returns the caption of the toggle button that needs to be shown below the button's
+     * title in case the state of the button is "on".
+     * In the case of the Volume widget, the caption message is shown only when the value is 0.
+     * @param {Number} value - The state of the button.
+     * @param {Object} messages - An object containing internationalizable messages for
+     * this component.
+     * @param {Component} that - The `gpii.qss.volumeButtonPresenter` instance.
+     * @return {String} The caption message for the toggle button.
+     */
+    gpii.qss.volumeButtonPresenter.getCaption = function (value, messages, that) {
+        that.changeButtonColor(value);
+        that.renderImage(value);
+        that.hideTitle(value);
+        return !value ? messages.caption : "";
+    };
+
+    /**
+     * Change the color of the "Volume & Mute" button if the value is 0.
+     * @param {jQuery} container - The jQuery container object
+     * @param {String} style - Contains css class
+     * @param {Boolean} value - The state of the button.
```

Comment:
Elsewhere this is described as Number. Be consistent throughout - and if it is a number, use an explicit test value === 0 rather than falsy